client/dashboard/domains/index.tsx

```diff
@@ -16,7 +17,9 @@ export function getDomainId( domain: Domain ): string {
 }
 
 function Domains() {
+	const { user } = useAuth();
 	const fields = useFields();
+	const actions = useActions( { user } );
```

Comment:
If `useActions` is a hook now, then can't is just call `useAuth` internally? Rather than expecting all of these call sites to grab the current user.